src/d_main.c

```diff
@@ -1983,6 +1985,15 @@ void D_DoomMain(void)
   //
 
   fastparm = clfastparm = M_CheckParm ("-fast");
+
+  //!
+  // @category game
+  // @vanilla
+  //
+  // Monsters have fast reflexes.
+  //
+
+  reflexparm = clreflexparm = M_CheckParm ("-reflex");
```

Comment:
I'm not a friend of the idea to add another (non-standard) command line parameter. I'd rather have a new "custom difficulty" submenu as discussed on Discord.